hacks.js

```diff
@@ -0,0 +1,17 @@
+// Addresses an issue where RTC peers are not properly garbage collected in Chromium:
+// https://bugs.chromium.org/p/chromium/issues/detail?id=825576
+// This error leads to WebTorrent Desktop and Chromium-based browsers being completely
+// unable to seed until the process is restarted:
+// https://github.com/webtorrent/webtorrent/issues/1981
+function forcePeerGC () {
+  queueMicrotask(() => {
```

Comment:
Why are you queuing a microtask in a timeout?  seems a little redundant.

Reply:
I'm a newbie. Trying to fix something I really don't understand very well.